app/views/commissions/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with(model: commission, local: true) do |form| %>
```

Comment:
How is this stuff being generated? A scaffold? We don't need a lot of these files, and our forms belong in the react component. In addition, I don't think that these routes are being routed to via `routes.rb`.